moveit_ros/benchmarks/src/BenchmarkExecutor.cpp

```diff
@@ -532,6 +532,11 @@ bool BenchmarkExecutor::plannerConfigurationsExist(const std::map<std::string, s
     planning_interface::PlannerManagerPtr pm = planner_interfaces_[it->first];
     const planning_interface::PlannerConfigurationMap& config_map = pm->getPlannerConfigurations();
 
+    // if the planner is chomp or stomp skip this function and return true for checking planner configurations for the
+    // planning group other wise an error occurs, this is kind of a trick which works well
```

Comment:
Even with `other wise` replaced with `otherwise`, I still can't parse this.  Why is this a trick? Perhaps the issue that with OMPL, a specific planning algorithm needs to be defined for a planning group, whereas with STOMP and CHOMP this is not necessary.

Reply:
changed comment